src/net-to-gross/netToGross.js

```diff
@@ -0,0 +1,55 @@
+const Big = require('big.js');
+const todayDate = new Date();
+const calculateTaxes = require('../taxCalculator.js');
+const insurancesCalcGross = require('./calcInsurance.js');
+const calculateTaxableIncome = require('./calcTaxableIncome.js');
+const insurancesCalculator = require('../insurancesCalculator.js');
+
+module.exports = async function netToGross(net, dependentCount = 0, region = 1, date = todayDate) {
+
+    const netBig = new Big(net)
+    if (netBig.lte(0)) {
+        return 0
+    }
+
+    const payslip = {};
+
+    //gross = net + totalTax + Insurances 
+    payslip.dependents = dependentCount;
+    payslip.dependentDeductionAmount = dependentCount * 4400000;
+
+    const taxableIncome = await calculateTaxableIncome(net, dependentCount)
+    const taxes = await calculateTaxes(taxableIncome);
+
+    payslip.totalTax = taxes.totalTax;
+    payslip.taxes = taxes.rates;
+
+    const insurance = await insurancesCalcGross(net, payslip.totalTax, region, date)
+    payslip.totalInsurance = insurance.totalInsurance
+    payslip.gross = insurance.grossFromIns;
+    payslip.afterInsurance = new Big(payslip.gross).minus(payslip.totalInsurance).toNumber();
+    payslip.netSalary = net;
+
+//Gross = net + totalTax + Insurance
+    const findInsuranceItem = insurancesCalculator(payslip.gross)
```

Comment:
a variable should be a noun. Pls find a better name instead of `findInsuranceItem`